Makefile

```diff
@@ -0,0 +1,51 @@
+BASE_IMG_NAME=kyma-dashboard
+REPO_IMG_DEV = $(DOCKER_PUSH_REPOSITORY)$(DOCKER_PUSH_DIRECTORY)/$(BASE_IMG_NAME)
+TAG = $(DOCKER_TAG)
+
+release-dev-web:
+	TARGET=web ENV=dev make build
+	TARGET=web ENV=dev make push
+
+release-dev-backend:
+	TARGET=backend ENV=dev make build
+	TARGET=backend ENV=dev make push
+
+release-dev-local:
+	TARGET=local ENV=dev make build
+	TARGET=local ENV=dev make push
+
+release-stage-web:
+	TARGET=web ENV=stage make build
+	TARGET=web ENV=stage make push
+
+release-stage-backend:
+	TARGET=backend ENV=stage make build
+	TARGET=backend ENV=stage make push
+
+release-stage-local:
+	TARGET=local ENV=dev make build
+	TARGET=local ENV=dev make push
+
+release-prod-web:
+	TARGET=web ENV=prod make build
+	TARGET=web ENV=prod make push
+
+release-prod-backend:
+	TARGET=backend ENV=prod make build
+	TARGET=backend ENV=prod make push
+
+release-prod-local:
+	TARGET=local ENV=dev make build
+	TARGET=local ENV=dev make push
```

Comment:
Maybe we can split these 9 jobs to 3 by building and pushing 3 images in one pipeline?